### PR TITLE
Fix deeplinks on mobile

### LINKED ISSUE #2554: Deep link does not open normally if the app is running on Android
**Describe the bug**
Deep link does not open normally if the app is running on Android, ios apps. 

**To Reproduce**
1. Open app
2. Click on Dex link
3. try to connect app with dex

**Expected behavior**

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: Android, ios 
 - OS: Android, ios

**Additional context**
Reproducible on dev branch.

### LINKED ISSUE #2687: Wallet doesn't react on opening invite deeplink on Android
For example, I have wallet settings opened. Then I switch to Chrome and navigate to a page with
```html
<a href="https://wallet.superhero.com/invite#2kFZ9jmH484gsFp7DGKAjchUuAA5vYUozWn2Ky79dm9Xqsubmm">
  invite link
</a>
```
After pressing on "invite link", the wallet gets opened again but on the settings page without showing anything like "invite claimed"/"invite already claimed"/"invalid invite link".

Alternatively, you can scan the QR code:

image

this bug was reproduced by @CedrikNikita

### PR DESCRIPTION
**Pull request by CedrikNikita**
fixes #2554,
fixes #2687

### CODE REVIEW

**Review by martinkaintas — APPROVED** [review]

**Comment by peronczyk** [comment]
Why those components has the `Loader` component imported now? Maybe the global `loading` state can be used?

**Comment by Liubov-crypto** [comment]
Fixed, LGTM

File: src/popup/router/index.ts [hunk 1]
```diff
@@ -149,12 +149,16 @@ if (IS_MOBILE_APP) {
...
-        const query = event.url.slice(prefix?.length).split('?')[1].split('&').reduce((acc, param) => {
-          const [key, value] = param.split('=');
-          return { ...acc, [key]: value };
-        }, {});
-
-        router.push({ path, query });
+        if (path.startsWith('/invite')) {
+          router.push({ path, hash: path.split('#')[1] });
+        } else {
+          const query = event.url.slice(prefix?.length).split('?')[1].split('&').reduce((acc, param) => {
```
Comment (peronczyk) [inline src/popup/router/index.ts:155]: This is the 2nd time where similar logic is used. I suggest to use: 

```
const [path, query] = ...split('?');
```

Or use methods available for `Url` class when instantiated on the path.

**Review by peronczyk — CHANGES_REQUESTED** [review]

**Comment by Liubov-crypto** [comment]
LGTM

File: src/popup/pages/InviteClaim.vue [hunk 2]
```diff
@@ -1,20 +1,35 @@
...
+} from '@ionic/vue';
+
+import { defineComponent } from 'vue';
 import { decode } from '@aeternity/aepp-sdk';
-import { useRouter, useRoute } from 'vue-router';
+import { useRoute } from 'vue-router';
 
 import { MODAL_CLAIM_GIFT_CARD } from '@/constants';
-import { useInvites, useModals } from '../../composables';
+import { useInvites, useModals, useUi } from '../../composables';
```
Comment (peronczyk) [inline src/popup/pages/InviteClaim.vue:17]:
```suggestion
import { useInvites, useModals, useUi } from '@/composables';
```

**Review by peronczyk — APPROVED** [review]

**Comment by Liubov-crypto** [comment]
Retested, LGTM

**Pull request merged**